dockstore-client/src/main/java/io/github/collaboratory/cwl/cwlrunner/CWLToolWrapper.java

```diff
@@ -41,15 +49,25 @@ public void checkForCWLDependencies() {
         String[] split = pair2.getKey().split(" ");
         final String schemaSaladVersion = split[split.length - 1].trim();
 
-        final String expectedCwltoolVersion = "1.0.20170828135420";
-        if (!cwlToolVersion.equals(expectedCwltoolVersion)) {
-            ArgumentUtility.errorMessage("cwltool version is " + cwlToolVersion + " , Dockstore is tested with " + expectedCwltoolVersion
+        ApiClient defaultApiClient;
```

Comment:
Nitpick: seems pointless to declare the variable and immediately assign to it on the next line. 
Just do both in one line